backend/tests/conftest.py

```
@@ -0,0 +1,77 @@
+    async with TestingSessionLocal() as session:
+        yield session
+
+
+@pytest.fixture
+async def client(db_session: AsyncSession) -> AsyncGenerator[AsyncClient, None]:
+    """
+    Fixture for async HTTP client.
+    Overrides the get_db dependency to use the test session.
+    """
+
+    async def override_get_db():
```

Ruff (0.14.7)

Missing return type annotation for private function `override_get_db`

(ANN202)